Framework/DataHandling/src/LoadIsawDetCal.cpp

```diff
@@ -271,11 +270,10 @@ void LoadIsawDetCal::exec() {
       ComponentScaling detScaling;
       detScaling.scaleX = CM_TO_M * width / det->xsize();
       detScaling.scaleY = CM_TO_M * height / det->ysize();
-      detScaling.compID = det->getComponentID();
+      detScaling.componentName = det->getFullName();
```

Comment:
Reading through the relevant methods in `ResizeRectangularDetector` it probably is compatible. But I did not intend to change this, so I will revert it.